Content.Server/Chemistry/ReagentEffects/PlantMetabolism/PlantAdjustHealth.cs

```diff
@@ -5,6 +5,11 @@ namespace Content.Server.Chemistry.ReagentEffects.PlantMetabolism
 {
     public sealed partial class PlantAdjustHealth : PlantAdjustAttribute
     {
+        public PlantAdjustHealth()
+        {
+            Attribute = "plant-attribute-health";
+        }
+
```

Comment:
in short: Don't do this.
Attribute and Positive should be `abstract` in the parent class, and overridden on these classes as needed. Don't add constants to the constructor of a sealed class.

Reply:
Not exactly the best with my grasp of C#, so point taken.

Reply:
This took me longer than it should have and I probably still did it wrong.

Reply:
I, uh, really expected I'd know how to do this, but guess there were hoops I didn't account for.